### PR TITLE
Update goyang to v0.2.8

### PR DESCRIPTION
**Pull request by wenovus**
- Updated test files as https://github.com/openconfig/goyang/issues/131 is now fixed.

### CODE REVIEW

File: exampleoc/opstateoc/update.sh [hunk 1]
```diff
@@ -35,7 +35,6 @@ go run ../../generator/generator.go -path=public,deps -output_file=oc.go \
   public/release/models/policy/openconfig-routing-policy.yang \
   public/release/models/lacp/openconfig-lacp.yang \
   public/release/models/system/openconfig-system.yang \
-  public/release/models/lldp/openconfig-lldp.yang \
```
Comment (robshakir) [inline exampleoc/opstateoc/update.sh:38]: Why was this removed please?

**Review by robshakir — CHANGES_REQUESTED** [review]
LGTM, but I want to understand why LLDP was removed.

File: exampleoc/opstateoc/update.sh (hunk 1 above)
Comment (wenovus) [reply exampleoc/opstateoc/update.sh:38]: Because it's already there (see last YANG file). There was a goyang change to error out for duplicate modules.

**Review by robshakir — APPROVED** [review]
Thanks!

File: exampleoc/opstateoc/update.sh (hunk 1 above)
Comment (robshakir) [reply exampleoc/opstateoc/update.sh:38]: Ah, apologies. I missed this.
